src/main/deltachat/webxdc.ts

```diff
@@ -200,6 +200,12 @@ export default class DCWebxdc extends SplitOut {
 
         const app_icon = icon_blob && nativeImage?.createFromBuffer(icon_blob)
 
+        const lastBounds = DesktopSettings.state.webxdcBounds?.[
+          `${accountId}.${msg_id}`
+        ] ?? {
```

Comment:
we should think about wether we want to store it in that file or inside of the accounts config table under a config key with the "ui" prefix:
- advantage would be that config.json does not know about message and account ids, same as we moved the selected chat id to a ui config key
- possible disadvantage would be that it's then exported on backups

Reply:
another reason is that I want to keep account specific values out of `config.json`, because my long term plan is to move the values there (global settings) to core account manager config keys (but the system for that does not exist yet) -> basically store the values of `config.json` in `accounts.toml` or a similar file that is managed by core instead of the runtime.

Anyways there is already a mechanism for account specific values with `ui.*` config keys, so let's use them.

Reply:
I'll merge this pr and then make a new one that switches it over to store it in core